console/src/components/UIKit/atoms/Tabs/Tabs.style.js

```diff
@@ -0,0 +1,85 @@
+import styled, { css } from 'styled-components';
+import {
+  typography,
+  border,
+  flexbox,
+  layout,
+  space,
+  color,
+} from 'styled-system';
+
+import { BaseStyledDiv } from '../Common.style';
+
+// ********************************* //
+
+const TabStyles = styled(BaseStyledDiv)``;
```

Comment:
?

Reply:
BaseStyledDiv is a Styled Component with some reusable styled-system functions and I'm extending it, So I don't have to import and apply styled-system functions when in the case of div element.